.circleci/config.yml

```diff
@@ -112,10 +109,9 @@ jobs:
       - yarn_install
       - vault/get-secrets:
           template-preset: semantic-release-ecosystem
-      # Lerna only looks for the name GH_TOKEN
```

Comment:
This should be already done in vault.

Reply:
You need the `vault/configure-lerna` step